pom.xml

```diff
@@ -61,7 +61,7 @@
     <akka.version>2.5.6</akka.version>
     <rat.version>0.12</rat.version>
     <maven.failsafe.version>2.19.1</maven.failsafe.version>
-    <grpc.version>1.11.0</grpc.version>
+    <grpc.version>1.14.0</grpc.version>
```

Comment:
Please don't change the code which is not related to the JIRA subject.
It's hard for us to track the change.